src/nupic/algorithms/Connections.cpp

```diff
@@ -41,6 +41,9 @@ Connections::Connections(CellIdx numCells,
                          SegmentIdx maxSegmentsPerCell,
                          SynapseIdx maxSynapsesPerSegment)
 {
+  // The current version number.
+  version_ = 1;
```

Comment:
This can probably just be a constant, right? We need to store it when we save the model but no need to store as an attribute in memory.